php/php.code.analysis/src/org/netbeans/modules/php/analysis/commands/CodingStandardsFixer.java

```diff
@@ -172,15 +172,17 @@ public void startAnalyzeGroup() {
     public List<Result> analyze(CodingStandardsFixerParams params, FileObject file) {
         assert file.isValid() : "Invalid file given: " + file;
         try {
-            
-            Integer result = getExecutable(Bundle.CodingStandardsFixer_analyze(analyzeGroupCounter++), findWorkDir(file))
+            File workDir = findWorkDir(file);
+            Integer result = getExecutable(Bundle.CodingStandardsFixer_analyze(analyzeGroupCounter++), workDir)
                     .additionalParameters(getParameters(params, file))
                     .runAndWait(getDescriptor(), "Running coding standards fixer..."); // NOI18N
             if (result == null) {
                 return null;
             }
-
-            return CodingStandardsFixerReportParser.parse(XML_LOG, file);
+            // if the project for the file is not found(i.e. if the workDir is not found),
+            // the results are not shown in the inspector window
+            FileObject root = workDir != null ? FileUtil.toFileObject(workDir) : file;
+            return CodingStandardsFixerReportParser.parse(XML_LOG, root);
```

Comment:
Just to be sure - isn't a problem that before it was `file` but now it can be a directory?

Reply:
I think it's OK. It just worked with the project directory (i.e. when the working directory is the project directory) before I fixed this. 
The case where the project directory(i.e. working directory) cannot be found: a user inspects a file or a directory from a Favorite pane. In such case, the API side doesn't show the result in the inspector window. But the XML result is shown in the Output window. I confirmed that.

Reply:
OK then. Thanks!